serialization/scala_signature.ksy

```diff
@@ -0,0 +1,993 @@
+meta:
+  id: scala_signature
+  endian: le
+  license: Apache-2.0
+  #imports:
+  #  - /common/vlq_base128_be
+  encoding: utf-8
+  xref:
+    wikidata: Q460584
```

Comment:
```suggestion
```

Please don't add arbitrary somewhat relevant topics to wikidata xref. It has to be exactly wikidata entry matching file format, or none.

Reply:
@GreyCat

> Please don't add arbitrary somewhat relevant topics to wikidata xref. It has to be exactly wikidata entry matching file format, or none.

Speaking of which, are these refs OK?

 - 
   _Domain Name System - hierarchical **distributed naming system** for computers, ..._ ([Q8767](https://www.wikidata.org/wiki/Q8767))
   
   https://github.com/kaitai-io/kaitai_struct_formats/blob/af44fefe99d5da3467679f01533f603734d15b58/network/dns_packet.ksy#L6

 - _Unified Extensible Firmware **Interface** - specification that defines a software interface between an operating system and platform firmware_ ([Q859398](https://www.wikidata.org/wiki/Q859398))
 
    https://github.com/kaitai-io/kaitai_struct_formats/blob/af44fefe99d5da3467679f01533f603734d15b58/security/efivar_signature_list.ksy#L5
 - _Logical Volume Manager - logical volume management **software** for Linux_ ([Q6667482](https://www.wikidata.org/wiki/Q6667482))
 
    https://github.com/kaitai-io/kaitai_struct_formats/blob/af44fefe99d5da3467679f01533f603734d15b58/filesystem/lvm2.ksy#L11
 - _Sound Manager_ [**software**] ([Q7564684](https://www.wikidata.org/wiki/Q7564684))

   https://github.com/kaitai-io/kaitai_struct_formats/blob/af44fefe99d5da3467679f01533f603734d15b58/macos/mac_os_resource_snd.ksy#L7
 - _Pickle - Python object serialization **library**_ ([Q7190889](https://www.wikidata.org/wiki/Q7190889))

   https://github.com/kaitai-io/kaitai_struct_formats/blob/af44fefe99d5da3467679f01533f603734d15b58/serialization/python_pickle.ksy#L10
 - _Windows Registry - hierarchical database that stores configuration settings ..._ ([Q463244](https://www.wikidata.org/wiki/Q463244))

   https://github.com/kaitai-io/kaitai_struct_formats/blob/af44fefe99d5da3467679f01533f603734d15b58/windows/regf.ksy#L7
- _Transport Layer Security - cryptographic protocols for securing data in transit_ ([Q206494](https://www.wikidata.org/wiki/Q206494))

  https://github.com/kaitai-io/kaitai_struct_formats/blob/af44fefe99d5da3467679f01533f603734d15b58/network/tls_client_hello.ksy#L5